test-resources/openapi-service-specs/test-service.json

```diff
@@ -26,9 +26,7 @@
   "paths": {
     "/entities": {
       "get": {
-        "tags": [
-          "entity"
-        ],
+        "tags": ["entity"],
```

Comment:
Since this has been a problem with the old openapi can you also include a real simple service with no schema at all and just a single get to see that also this leads to a compiling example.

Reply:
Sure

Reply:
done.